test/fixtures/mongodb/data-default.js

```diff
@@ -104,6 +107,7 @@ module.exports = {
     {
       _id: ObjectId("5bb6540cbecb4e208da0fb64"),
       name: "Health and Wellness",
+      pronunciation: "",
```

Comment:
pronunciation is an optional field, right? we should only specify it where it has a value